### PR TITLE
return array repr as Tensor repr

### PR DESCRIPTION
**Pull request by gaxler**
As discussed in https://github.com/google/TensorNetwork/issues/814#issue-694297329

__repr__ behavior is to fall back on the underlying array __repr__

@mganahl @Thenerdstation Could you please link the issue to the PR?

### CODE REVIEW

**Comment by gaxler** [comment]
@googlebot I signed it!

**Review by Thenerdstation — APPROVED** [review]

**Comment by mganahl** [comment]
Hey @gaxler, once the build is fixed we can merge this!

**Comment by alewis** [comment]
Sorry to barge in at the end here, but is the __repr__ going to be actually indistinguishable from that of the underlying array? I think it should also tell you the object is a tn.Tensor.

File: tensornetwork/tensor.py [hunk 1]
```diff
@@ -185,3 +185,6 @@ def __matmul__(self, other: "Tensor") -> "Tensor":
       raise ValueError(errstr)
     array = self.backend.matmul(self.array, other.array)
     return Tensor(array, backend=self.backend)
+
+  def __repr__(self):
+    return self.array.__repr__()
```
Comment (alewis) [inline tensornetwork/tensor.py:190]: Could we do something like
`return "tn.Tensor: \" + self.array.__repr__()`?

Comment (mganahl) [reply tensornetwork/tensor.py:190]: SGTM!

**Pull request closed without merging**